test/scripts/e2e.sh

```diff
@@ -125,9 +125,10 @@ if [ -z "$E2E_TEST_FILTER" ] || [ "$E2E_TEST_FILTER" == "SCRIPTS" ]; then
     . "${TEMPDIR}/ve/bin/activate"
     "${TEMPDIR}/ve/bin/pip3" install --upgrade pip
 
-    # Pin a version of our python SDK's so that breaking changes don't spuriously break our tests.
-    # Please update as necessary.
-    "${TEMPDIR}/ve/bin/pip3" install py-algorand-sdk==1.17.0
+    # Pin major version of our python SDK's so that breaking changes
+    # don't spuriously break our tests.  If a minor version breaks our
+    # tests, we ought to find out.
+    "${TEMPDIR}/ve/bin/pip3" install py-algorand-sdk==2
```

Comment:
> The use of == (without at least the wildcard suffix) when defining dependencies for published distributions is strongly discouraged as it greatly complicates the deployment of security fixes.

(from the version [spec](https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers)

So I assume `==2` means `2.0.0`, is it the intent? If it is not then probably `== 2.*` is a better alternative ?

Reply:
Hmm, yeah, I guess that's right.  Will fix!  Thanks.

Reply:
Actually, I suppose I want ">=2"?

Reply:
I'll use 2.* since I don't want 3.